README.md

```diff
@@ -131,6 +133,7 @@ The full description of supported registers is available in the API documentatio
 
 If the available registers define more than one transport configuration, a redundant transport will be initialized.
 
+<<<<<<< Updated upstream
```

Comment:
Please always review your changesets before submission, it really saves time for everyone involved.